lib/InstantSearch.js

```diff
@@ -45,6 +51,7 @@ Usage: instantsearch({
     }
 
     let client = algoliasearch(appId, apiKey);
+    client.addAlgoliaAgent('instantsearch_' + version);
```

Comment:
Could it rather use a space instead of the `_`, we're parsing it here and there so it's easier to match the current regexp :)

I would go for `'instantsearch.js ' + version`.